selfdrive/car/hyundai/values.py

```diff
@@ -28,7 +28,7 @@ class CAR:
   KONA = "HYUNDAI KONA 2020"
   KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
   SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019"
-  SONATA = "HYUNDAI SONATA 2020"
+  SONATA = "HYUNDAI SONATA 2020-2021"
```

Comment:
We don't update the years in the version string. Feel free to update the year in the README.md though!

Reply:
Sounds good! Should I edit this PR / resubmit another PR? Or will these lines get modified before merge?

Reply:
You can just edit this PR.

Reply:
Edited this PR to keep line 31 unchanged from commaai source